src/kernels/installer/moduleInstaller.node.ts

```diff
@@ -184,27 +185,3 @@ export abstract class ModuleInstaller implements IModuleInstaller {
         flags?: ModuleInstallFlags
     ): Promise<ExecutionInstallArgs>;
 }
-
```

Comment:
Move this to a non node specific file so errorHandler didn't have to be node specific anymore.